util/topgen.py

```diff
@@ -526,6 +527,40 @@ def generate_ac_range_check(topcfg: Dict[str, object], out_path: Path) -> None:
     ipgen_render("ac_range_check", topname, params, out_path)
 
 
+# Generate RACL collateral
+def generate_racl(topcfg: Dict[str, object], out_path: Path) -> None:
+    # Not all tops use RACL
+    if 'racl_config' not in topcfg:
+        return
+
+    topcfg['racl'] = parse_racl_config(topcfg['racl_config'])
+
+    log.info('Generating RACL Control IP with ipgen')
+    topname = topcfg['name']
+
+    for racl_group, policies in topcfg['racl']['policies'].items():
+        params = {
+            "nr_racl_bits": 4,
+            "nr_policies": len(policies),
+            "policies": policies
+        }
+
+        # If we have more RACL policy groups, uniquify the control IP
+        # if len(topcfg['racl']['policies']) > 1:
+        #     params['module_instance_name'] = f'racl_ctrl_{racl_group}'
+
+        # # Only render the RACL groups that are really instantiated in that top
+        # for m in topcfg['module']:
+        #     if m['name'] == params['module_instance_name']:
+        #         ipgen_render("racl_ctrl", topname, params, out_path)
+        #         break
+        # TODO: The obove code, would be the correct if ipgen correctly supports rendering multiple
```

Comment:
Do we already have an issue to track this limitation? If yes, please link it. If no, please create one and then link it :-)

Reply:
Done https://github.com/lowRISC/opentitan/issues/25673